device/nokia/arm64-nokia_ixs7215_52xb-r0/platform.json

```diff
@@ -227,16 +257,16 @@
                 "name": "Ethernet47"
             },
             {
-                "name": "Ethernet48"
+                "name": "SFP48"
```

Comment:
why such changes?

Reply:
On this box the first 48 ports are RJ45 and the last 4 ports are SFP+ ports.
This is done to differentiate this.
We have the same notation for 7215-IXS-T1(M0) platform as well, so was trying to be consistent.